packages/tokens/style-dictionary/build.ts

```diff
@@ -0,0 +1,43 @@
+import StyleDictionary from "style-dictionary";
```

Comment:
Ideally, all the source files should be located in a "src" folder. 

your project should look like: 

- pnpm-lock.yarm
- package.json
- .eslintrc.json
- src/
    - style-dictionary
    - tokens
- test/ if you had some.
- build/ for your intermediary js files
- dist/ for files you want to distribute